lib/src/api_impl/stderr_logging_impl.dart

```diff
@@ -16,14 +16,14 @@ class StderrRequestLoggingImpl extends LoggingImpl {
   final DateTime _startTimestamp = DateTime.now().toUtc();
   final List<_LogLine> _gaeLogLines = <_LogLine>[];
 
-  LogLevel _currentLogLevel;
+  LogLevel _currentLogLevel = LogLevel.DEBUG;
```

Comment:
Nit: Is this the default now? I'd expect info.

Reply:
Nothing new, this was already the case in `_resetState()`, which is called in the constructor.

This just avoids it ever being `null`, even if it never can be :D